src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/XamlToRtfWriter.cs

```diff
@@ -3672,11 +3672,11 @@ ref StrikeState strikeState
             {
                 ulState = ULState.ULNone;
                 strikeState = StrikeState.StrikeNone;
-                if (decoration.IndexOf("Underline", StringComparison.OrdinalIgnoreCase) != -1)
+                if (decoration.Contains("Underline"))
```

Comment:
Dose it change the behavior?

Reply:
Thank you for bringing it up. I have addressed the issue in the latest commit.